AmazonChimeSDKDemo/AmazonChimeSDKDemo/Base.lproj/Main.storyboard

```diff
@@ -898,6 +906,55 @@
             </objects>
             <point key="canvasLocation" x="1853.6231884057972" y="112.5"/>
         </scene>
+        <!--Test Settings View Controller-->
```

Comment:
Can we make sure all the new UI elements have accessibility identifier in case we need to find those elements in UI test?

Reply:
Done.